research/picai/preprocessing.py

```diff
@@ -120,12 +120,14 @@ def __init__(
         Class representing a case from the PICAI dataset.
 
         scan_paths filenames are assumed to have the following format: <patient_id>_<study_id>_<modality>.mha
-        where modality is a three letter string of ['t2w', 'adc', 'hbv']
+        where modality is a three letter string of ['t2w', 'adc', 'hbv']. Assumes the order of scan paths is
+        t2w, adc and hbv.
```

Comment:
As discussed briefly in the call, I think we can make this comment slightly more clear in the sense that the order of the scan paths doesn't necessarily have to follow a specific structure but it MUST be consistent with the ordering of `self.scans` throughout. So something like 
```
where modality is a three letter string of ['t2w', 'adc', 'hbv']. NOTE: the ordering self.scan_path and self.scans must remain consistent.
```